docker-compose.yml

```diff
@@ -11,7 +11,7 @@ services:
     volumes:
       - dbdata:/var/lib/postgresql/data
     ports:
-      - 5433:5432 # Mapped to 5433 in case Postgres is already running locally on 5432
+      - 8081:5432 # Mapped to 8081 in case Postgres is already running locally on 5432
```

Comment:
This port is free on Windows and Mac machines (5433 wasn't on Sam's Mac), so just makes life easier changing it for all